src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -3716,6 +3749,9 @@ private void handlePostGenericSubObj(RoutingContext ctx, String subObj) {
           body = JsonObject.mapFrom(ctx.getBodyAsJson().mapTo(getSubObjClass(subObj))).put(ID, UUID.randomUUID().toString());
           respBody = body.encodePrettily();
           break;
+        case 400:
```

Comment:
Wouldn't invalid JSON result in a 422?

Reply:
I was guided by the error codes returned from the orders-storage. Storage can return errors 400, 401, 500 according to the API doc. Craig, let me know if we need to change this.

Reply:
Taking into account `@Validate` annotation added to `postOrdersPieces` method, the `400` error might only happen if mod-orders and mod-orders-storage modules work with different schemas...

Reply:
I believe you're right @piotr-kalashuk.